documentation/staging/content/navigate/kubernetes/k8s-ingress-controller.md

```diff
@@ -86,7 +86,9 @@ When editing a route:
 - Use the `Name` field to set the route name.  
 - Use the `Virtual Host` and `Path Expression` fields to define the matching rules that determine which requests match this route.  
 - All requests in the defined rules are routed to the service specified by the `Target Service` field that resides in the namespace specified by the read-only
-`Target Service Namespace` field and the port specified by the `Target Port` field. Once the domain has been deployed, you can also select the values from the drop down list. 
+`Target Service Namespace` field and the port specified by the `Target Port` field. Once the domain has been deployed, you can select the `Target Service`
+ value from the drop-down list of available service in the WebLogic Kubernetes Operator domain's namespace. Once the `Target Service` is selected,
```

Comment:
available service -> available services
Once the `Target Service` is selected -> After the `Target Service` is selected